Docs/Getting-Started-STT_and_TTS.md

```
@@ -0,0 +1,337 @@
+  -H "X-API-KEY: $SINGLE_USER_API_KEY"
+# Then send base64-encoded audio chunks per the server protocol
+```
+
+Troubleshooting
+- Long files: prefer shorter clips or chunk client-side.
+- Out-of-memory: try a smaller model (e.g., `whisper-medium`), or run on GPU.
+
+---
+
+## Verifying Setup via WebUI
+- Open http://127.0.0.1:8000/webui/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)